external/fv3fit/fv3fit/emulation/scoring.py

```diff
@@ -53,28 +41,27 @@ def score(target, prediction) -> ScoringOutput:
 
 
 def score_single_output(
-    target: np.ndarray, prediction: np.ndarray, name: str, rescale: bool = True
+    target: np.ndarray, prediction: np.ndarray, name: str
 ) -> ScoringOutput:
     """
     Score a single named output from an emulation model. Returns
     a flat dictionary with score/profile keys preceding variable
     name for Weights & Biases chart grouping.
 
+    Precipitation scores are rescaled for backwards compatibility
+
     Args:
         target: truth values
         prediction: emulated values
         name: name of field being emulated to be added to the key
             of each score
-        rescale: rescale outputs using factor in top-level
-            SCALE_VALUES mapping if available
     """
 
-    if rescale:
-        try:
-            target *= SCALE_VALUES[name]
-            prediction *= SCALE_VALUES[name]
-        except KeyError:
-            logger.error(f"No scaling value found for {name}. Leaving unscaled...")
+    # including for backwards compatibility for precip scores
+    if name == "total_precipitation":
+        precip_scaling = (1000 / (900 / (3600 * 24)),)  # mm / day
```

Comment:
Is the factor of 1000 correct? If the original units are kg/m2/s then the liquid water equivalent is 

1 [kg/m^2/s] / 1000 [kg/m^3] = 1  / 1000 [m / s] = 1 [mm/s] = 86400 [mm/d]

Nice to use variable names to name some of these convesion factors:

```
timestep = 900
seconds_in_day = 86400
density_liquid_water = 1000
mm_in_m = 1000
precip_scaling = mm_in_m / density_in_liquid_water * seconds_in_day / timestep
```

Reply:
```! rain1 has units m, and represents surface precip over dtp```

I'll update the scaling to indicate the units more clearly.